src/poliastro/core/propagation/cowell.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+
+from poliastro._math.ivp import DOP853, solve_ivp
+from poliastro.core.propagation.base import func_twobody
+
+
+def cowell(k, r, v, tofs, rtol=1e-11, *, events=None, f=func_twobody):
```

Comment:
This is essentially like the old function, but I moved it to `core` and stripped it from unit handling.